### DISCUSSION TITLE
Cannot run Chia 1.2 on Mac OS intel gives malicious software error

### DISCUSSION THREAD

**Discussion by cvee72** [Chia-Network/chia-blockchain#7190] [Support]
Hi,

Just installed Chia 1.2 and trying to run it gives message :-
“Chia” can’t be opened because Apple cannot check it for malicious software.

It doesn't give me option in system settings to run anyway.
So i cannot run it at all?

Anyone else on mac see this problem?

**Comment by joelrb** [comment]
Go to your Mac's Preferences > Security & Privacy > General tab.. look down to Allow apps downloaded from:

Apple flags any software as potentially malicious that does not come from the App Store.

**Reply by cvee72** [reply]
I already have the option for :-
App Store and identified developers

Still doesn't allow me to run it... NOR does it give me the option to bypass this issue as you suggested in the other post.

**Reply by joelrb** [reply]
It doesn't show the "Open Anyway" button? 

Something like this?
![open-anyway](https://user-images.githubusercontent.com/225284/124884319-bf5d8400-e004-11eb-8458-ddbbdc6f776c.jpg)

**Comment by cvee72** [comment]
No it doesn’t.
That’s why it’s weird.

If I could open anyway… I wouldn't have raised this query.

I’m stumped.

**Reply by joelrb** [reply]
Try this last resort to open app from Anywhere.

Open Terminal and enter the following code to get your Anywhere option:

sudo spctl --master-disable

It will show the option:
![how_to_allow_apps_from_anywhere_mac_780b](https://user-images.githubusercontent.com/225284/124888148-698adb00-e008-11eb-9793-4d283feac7d2.jpg)

After Chia is opened. Just remove the option again in the Terminal with:

sudo spctl --master-enable

**Comment by cvee72** [comment]
Ok... that workaround worked!!!!
Launched Chia now.

Thanks.

**Comment by loppefaaret** [comment]
we had some issues with a macos installers - you could redownload and reinstall

**Comment by BrandtH22** [comment]
Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
